contracts/Core/Parameters.sol

```diff
@@ -128,4 +139,12 @@ contract Parameters is IParameters, ACL {
     function getStakeModifierHash() external pure override returns (bytes32) {
         return _STAKE_MODIFIER_HASH;
     }
-}
+
+    function getStakeRegulatorHash() external pure override returns (bytes32) {
+        return _STAKE_REGULATOR_HASH;
+    }
+
+    function getRewardPoolModifierHash() external pure override returns (bytes32) {
+        return _REWARD_POOL_MODIFER_HASH;
+    }
+}
```

Comment:
It is generally a good practise to end files with new line - https://stackoverflow.com/a/729795

Reply:
Yep will do

Reply:
Done https://github.com/razor-network/contracts/pull/121/commits/e61016b7ecbf64a54dbbd3db02332c2addcd06af